components/enedis/actions/get-address/get-address.mjs

```diff
@@ -0,0 +1,23 @@
+import enedis from "../../enedis.app.mjs";
+
+export default {
+  type: "action",
+  key: "enedis-get-address",
+  version: "0.0.1",
+  name: "Get address",
+  description: "Returns the address of a client.",
+  props: {
+    enedis,
+    usage_point_id: {
+      propDefinition: [
+        enedis,
+        "usage_point_id",
```

Comment:
The main code should use `camelCase` capitalization - even if you have to later transform it into a `snake_case` property when making the request, if the API expects the parameter named as such.
```suggestion
    usagePointId: {
      propDefinition: [
        enedis,
        "usagePointId",
```